packages/app-tiles/src/AppTiles.stories.tsx

```diff
@@ -1,83 +1,76 @@
 import React from 'react';
-import { Story, Meta } from '@storybook/react';
+import { StoryObj } from '@storybook/react';
 import AppTiles from './AppTiles';
 
 export default {
   title: 'Components/AppTiles',
-  parameters: {
-    docs: {
-      description: {
```

Comment:
please add back component description

Reply:
you can leave it in the meta params or switch to documentation comments https://storybook.js.org/docs/react/api/doc-block-description#writing-descriptions